po/it.po

```diff
@@ -245,7 +242,9 @@ msgstr "%N %V - Schermate"
 msgid ""
 "%N %V - %?F?Feeds&Your feeds? (%u unread, %t total)%?F? matching filter `"
 "%F'&?%?T? - tag `%T'&?"
-msgstr "%N %V - I tuoi feed (%u non letti, %t totali)%?T? - tag `%T'&?"
+msgstr ""
+"%N %V - %?F?Feeds&YI tuoi feed (%u non letti, %t totali)%?F? corrispondenti al filtro `"
```

Comment:
Just checking: is "YI" a typo here, should it be "I" instead? (I don't know Italian, so totally guessing here.)

Reply:
Yep, bad keypress, good catch, I didn't notice it.
Fixing it.